cli/cluster/errors.go

```diff
@@ -48,7 +48,7 @@ func ErrorFailedToConnectOperator(originalError error, envName string, operatorU
 		originalErrMsg = urls.TrimQueryParamsStr(errors.Message(originalError)) + "\n\n"
 	}
 
-	msg := fmt.Sprintf("%sunable to connect to your cluster in the %s environment (operator endpoint: %s)\n\n", originalErrMsg, envName, operatorURL)
+	msg := fmt.Sprintf("%s unable to connect to your cluster in the %s environment (operator endpoint: %s)\n\n", originalErrMsg, envName, operatorURL)
```

Comment:
I believe it is correct without the space, since on line 48, `\n\n` is appended to the `originalErrMsg` string.

If you think it's more clear, we could switch it to:

```go
msg := originalErrMsg + fmt.Sprintf("unable to connect to your cluster in the %s environment (operator endpoint: %s)\n\n", envName, operatorURL)
```

What do you think?

Reply:
> I believe it is correct without the space, since on line 48, \n\n is appended to the originalErrMsg string.

SGTM